website/versioned_docs/version-29.1/MockFunctions.md

```diff
@@ -17,27 +17,33 @@ function forEach(items, callback) {
     callback(items[index]);
   }
 }
+module.exports = forEach;
 ```
 
 To test this function, we can use a mock function, and inspect the mock's state to ensure the callback is invoked as expected.
 
 ```javascript
```

Comment:
I can’t use a suggestion, add the `title` attribute to the code block above: `js title="foreach.js"`. Just to show that this is the file being imported.

And add one here too:

```suggestion
```js title="foreach.test.js"
```